xtos/include/sof/lib/cpu.h

```diff
@@ -13,6 +13,7 @@
 
 #ifndef __SOF_LIB_CPU_H__
 #define __SOF_LIB_CPU_H__
+#ifndef __ZEPHYR__
```

Comment:
Any suggestions what to do with this file? Zephyr build includes files from _xtos/include/sof/lib_.

Reply:
@tmleman these are in transition, they are being separated one by one. End result will be the Zephyr config wont include any xtos headers.